src/server/api/routes/qr.router.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+const qrController = require('../controllers/qr.controller');
+
+/**
+ * @swagger
+ * /qr/{GAME}:
+ *  post:
+ *    summary: insert a game instance for a user and a game and recieve its qr code
+ *    description:
+ *      Will return the qr code for game instance.
+ *    produces: application/json
+ *    parameters:
+ *
+ *     - in: path
+ *       name: GAME
```

Comment:
You can add a description here to inform the user what kind of data they should expect?

Reply:
done